wallet/src/main/java/ch/admin/bag/covidcertificate/wallet/detail/CertificateDetailItem.kt

```diff
@@ -26,42 +30,46 @@ data class TitleItem(@StringRes val titleResource: Int) : CertificateDetailItem(
 
 	override fun bindView(view: View) {
 		view.findViewById<TextView>(R.id.item_title).setText(titleResource)
+		view.findViewById<TextView>(R.id.item_title_english).text = getEnglishTranslation(view.context, titleResource)
 	}
 }
 
-data class TitleStatusItem(@StringRes val labelResource: Int, val status: String) : CertificateDetailItem() {
-	companion object {
-		const val layoutResource = R.layout.item_detail_title_status
-	}
-
-	override fun bindView(view: View) {
-		view.findViewById<TextView>(R.id.item_title_status_label).setText(labelResource)
-		view.findViewById<TextView>(R.id.item_title_status_value).text = status
-	}
-}
-
-data class ValueItem(@StringRes val labelResource: Int, val value: String) : CertificateDetailItem() {
+data class ValueItem(@StringRes val labelResource: Int, val value: String, val isTranslatedToEnglish: Boolean = true) :
+	CertificateDetailItem() {
 	companion object {
 		const val layoutResource = R.layout.item_detail_value
 	}
 
 	override fun bindView(view: View) {
 		view.findViewById<TextView>(R.id.item_value_label).setText(labelResource)
 		view.findViewById<TextView>(R.id.item_value_value).text = value
+		val englishLabel = view.findViewById<TextView>(R.id.item_value_label_english)
+		englishLabel.isVisible = isTranslatedToEnglish && view.context.getString(R.string.language_key) != "en"
```

Comment:
Maybe it would be useful to pull this out into an extension function (since it's used in the Builder a few times as well):
```kotlin
fun Context.isLocaleEnglish() =  this.getString(R.string.language_key) != "en"
```